contracts/Version.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+abstract contract Version {
+    string public constant VERSION = "v0.0.1";
```

Comment:
suggestion: this should work with upgradable contract, but definitely needs verifying with a test.

When you add the version to one of the upgradable bond classes, adding a case in the `.test.ts` would be excellent 😁

Reply:
Oh the proxies are upgradeable proxies! Won't the version be bumped? say: `v0.0.1` -> `v0.0.2`? 

(Ill add a test.)

Reply:
Yeah, I'd expect the version to be bumped (and that'll be enforced by the GitHub actions you're writing too 😉 )